site2/website/versioned_docs/version-2.8.1/client-libraries-java.md

```diff
@@ -122,6 +122,9 @@ int|`connectionTimeoutMs`|Duration of waiting for a connection to a broker to be
 int|`requestTimeoutMs`|Maximum duration for completing a request |60000
 int|`defaultBackoffIntervalNanos`| Default duration for a backoff interval | TimeUnit.MILLISECONDS.toNanos(100);
 long|`maxBackoffIntervalNanos`|Maximum duration for a backoff interval|TimeUnit.SECONDS.toNanos(30)
+SocketAddress|`socks5ProxyAddress`|Socks5 proxy address | None
```

Comment:
```suggestion
SocketAddress|`socks5ProxyAddress`|SOCKS5 proxy address | None
```

Please pay attention to the capitalization of the proper noun: https://en.wikipedia.org/wiki/SOCKS

Please correct all occurrences, thanks.